lib/invoice_item_repository.rb

```diff
@@ -121,4 +121,26 @@ def find_invoice(id)
   def find_item(item_id)
     sales_engine.find_item_by_id(item_id)
   end
+
+  def create_new_items(items, id)
+    items.each do |item|
+      grouped_items = items.group_by do |item|
+        item
+      end
+      quantity = grouped_items.map do |item|
+        item.count
+      end.uniq.flatten.join
+      line = {
+        id:         "#{invoice_items.last.id + 1}",
+        item_id:    item.id,
+        invoice_id: id,
+        quantity:   quantity,
+        unit_price: item.unit_price,
+        created_at: "#{Date.new}",
+        updated_at: "#{Date.new}"
+              }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.